internal/handlers/irc/irc.go

```diff
@@ -14,14 +13,15 @@ and the IRCSettings that were passed into NewClient
 type Client struct {
 	*girc.Client
 	Settings internal.IRCSettings
+	verbose internal.Debug
```

Comment:
Once you define that interface, please change structs to contain an `internal.DebugLogger` (or whatever you call it) instead of `internal.Debug`